src/components/NewCommentForm.tsx

```diff
@@ -59,11 +59,13 @@ export const NewCommentForm: React.FC<Props> = ({ onSubmit }) => {
 
     setSubmitting(true);
 
+    setTimeout(() => setSubmitting(false), 500);
```

Comment:
The use of `setTimeout` here is a bit misleading, it gives the impression that some asynchronous operation is happening, but it's just a delay. If you are trying to simulate a network request, you should indicate that in a comment.